src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Controllers/Api/Statistics/BoundaryLevelControllerTests.cs

```diff
@@ -0,0 +1,178 @@
+
+    [Fact]
+    public async Task GetBoundaryLevel_Success()
+    {
+        await TestApp.AddTestData<StatisticsDbContext>(context =>
+        {
+            context.BoundaryLevel.Add(
+                new()
+                {
+                    Id = 1,
+                    Created = DateTime.Now,
+                    Label = "Boundary Level 1",
+                    Level = GeographicLevel.Country,
+                    Published = DateTime.Now,
+                });
+        });
+
+        var client = TestApp
+            .SetUser(AuthenticatedUser(RoleClaim(RoleNames.BauUser)))
+            .CreateClient();
+
+        var response = await client.GetAsync("api/boundary-level/1");
+
+        var result = response.AssertOk<BoundaryLevelViewModel>();
+
+        Assert.Multiple(
+            () => Assert.Equal(1, result.Id),
+            () => Assert.Equal("Boundary Level 1", result.Label)
+        );
+    }
+
+    [Fact]
+    public async Task UpdateBoundaryLevel_Success()
+    {
+        await TestApp.AddTestData<StatisticsDbContext>(context =>
+        {
+            context.BoundaryLevel.Add(
+                new()
+                {
+                    Id = 1,
+                    Created = DateTime.Now,
+                    Label = "Boundary Level 1",
+                    Level = GeographicLevel.Country,
+                    Published = DateTime.Now,
+                });
+        });
+
+        var client = TestApp
+            .SetUser(AuthenticatedUser(RoleClaim(RoleNames.BauUser)))
+            .CreateClient();
+
+        var request = new BoundaryLevelUpdateRequest
+        {
+            Id = 1,
+            Label = "Updated Boundary Level 1",
+        };
+
+        var response = await client.PatchAsync("api/boundary-level", JsonContent.Create(request));
+
+        response.AssertNoContent();
+
+        await using var context = TestApp.GetDbContext<StatisticsDbContext>();
+
+        var saved = await context.BoundaryLevel.SingleAsync(bl => bl.Id == request.Id);
+
+        Assert.Multiple(
+            () => Assert.Equal(request.Id, saved.Id),
+            () => Assert.Equal(request.Label, saved.Label),
+            () => Assert.NotNull(saved.Updated)
+        );
+    }
+
+    [Fact]
+    public async Task CreateBoundaryLevel_Success()
+    {
+        await TestApp.AddTestData<StatisticsDbContext>(context =>
+        {
+            context.BoundaryLevel.Add(
+                new()
+                {
+                    Id = 1,
+                    Created = DateTime.Now,
+                    Label = "Boundary Level 1",
+                    Level = GeographicLevel.Region,
+                    Published = DateTime.Now,
+                });
+        });
+
+        var client = TestApp
+            .SetUser(AuthenticatedUser(RoleClaim(RoleNames.BauUser)))
+            .CreateClient();
+
+        var path = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location)!, @"REG - Regions England BUC 202212.geojson");
```

Comment:
I'd move the geojson file into the `Resources` directory. This is the pattern we use in other places - see `DataArchiveValidationServiceTests#CreateFormFileFromResource`